lib/memcached/behaviors.rb

```diff
@@ -1,6 +1,22 @@
 module Memcached
   module Behaviors
+    def set_behavior(behavior, value)
+      behavior = convert_behavior(behavior)
+      value = convert_value(behavior, value)
+      _set_behavior(behavior, value)
+    rescue Memcached::Deprecated
+      warn "Behavior #{behavior_string} is deprecated, and won't work anymore."
```

Comment:
`behavior_string` is undefined here.
We probably should add a test case for this Deprecation warning.